app/views/overview/_service-binding.html

```diff
@@ -1,20 +1,33 @@
-<div class="row service-binding">
-  <div class="col-sm-5 col-md-6">
-    <h3 ng-if="$ctrl.serviceClass">
-      {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
-      <small>{{$ctrl.binding.spec.instanceRef.name}}</small>
-    </h3>
-    <h3 ng-if="!$ctrl.serviceClass">
-      {{$ctrl.binding.spec.instanceRef.name}}
-    </h3>
+<div class="service-binding">
+  <div class="row">
+    <div class="col-sm-5 col-md-6">
+      <h3 ng-if="$ctrl.serviceClass">
+        <span ng-if="$ctrl.serviceClass">
+          {{$ctrl.serviceClass.externalMetadata.displayName || $ctrl.serviceClass.metadata.name}}
+        </span>
+        <span ng-if="!$ctrl.serviceClass">
+          {{$ctrl.binding.spec.instanceRef.name}}
+        </span>
+        <small ng-if="$ctrl.serviceClass">{{$ctrl.binding.spec.instanceRef.name}}</small>
+        <small>created <span am-time-ago="$ctrl.binding.metadata.creationTimestamp"></span></small>
+      </h3>
+    </div>
+    <div class="col-sm-7 col-md-6" ng-if="!($ctrl.binding | isBindingReady)">
+      <status-icon status="'Pending'"></status-icon>Pending
+    </div>
   </div>
-  <div class="col-sm-7 col-md-6 overview-bindings">
-    <span ng-if="!($ctrl.binding | isBindingReady)">
-      <status-icon status="'Pending'"></status-icon> Pending
-    </span>
-    <a
-      ng-if="($ctrl.binding | isBindingReady) && ('secrets' | canI : 'get')"
-      ng-href="{{$ctrl.secrets[$ctrl.binding.spec.secretName] | navigateResourceURL}}">
+  <div class="service-binding-actions" ng-if="true || ($ctrl.binding | isBindingReady) && !ctrl.binding.metadata.deletionTimestamp">
```

Comment:
That seems to be the intent. Are you seeing the secret created before the binding is ready?
I can't decide if we should show it since it presumably won't "work" until ready.

Any errors in the service catalog controller-manager log? Would be good to find out why they're getting stuck for you.